common/src/node/ethereum/provider.rs

```diff
@@ -0,0 +1,35 @@
+use crate::EthError;
+use ethers::providers::{Http, Provider};
+use url::Url;
+
+#[cfg(not(target_arch = "wasm32"))]
+static DEFIWALLETCORE_AGENTINFO: Option<&'static str> = option_env!("DEFIWALLETCORE_AGENTINFO");
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
what's a difference for the method on wasm32 vs normal, their content looks identical?

Reply:
set_agent not supported for wasm32 target yet

Reply:
maybe can feature guard just that line/block, not the whole function?

Reply:
yes , i'll check